packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,6 +402,19 @@ class EditableText extends StatefulWidget {
   /// Defaults to EdgeInserts.all(20.0).
   final EdgeInsets scrollPadding;
 
+  /// {@template flutter.widgets.editableText.enableInteractiveSelection}
+  /// If true, then long-pressing this TextField will select text and show the
+  /// cut/copy/paste menu, and tapping will move the text caret.
+  ///
+  /// True by default.
+  ///
+  /// If false, most of the accessibility support for selecting text, copy
+  /// and paste, and moving the caret will be disabled.
+  ///
+  /// If false, the caret will remain at the end of the [controller]'s text.
```

Comment:
@HansMuller is this always guaranteed? For example, would it be true if the user changes the value of the TextEditingController with a selection value that's not collapsed at the end?

Reply:
The user can't create a selection (with long-press) or paste.

Reply:
Sorry, I meant to say if a developer changes the value.

Reply:
This feature is only intended to limit what the mobile user can do. Developers can use the TextEditingController to change the selection.  

I've removed the "the caret will remain at the end..." comment since it didn't seem to improve the property description.

Currently if (on Android) if the user is using a hardware keyboard they'll be able to move the cursor, create a selection, copy and paste.  I could disable that as well, if that's necessary?

Reply:
That would be useful for my team's use case.

My teammate noticed he could change the cursor position in his iOS emulator as well (without this change).

Reply:
I don't think this is a blocker for us though. We are probably OK with it being done later.

Reply:
I've opened an issue for this: https://github.com/flutter/flutter/issues/23248